src/index.js

```diff
@@ -1,14 +1,43 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 import '@fortawesome/fontawesome-free/js/all.js';
 
-function component() {
-  const element = document.createElement('div');
+// function component() {
+//   const element = document.createElement('div');
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history.